js-intro.js

```diff
@@ -36,16 +61,42 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+    onlyVowels = (array) => {
+    var vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+    return array
+    .split('')
+    .filter(char => !vowels.includes(char))
+    .join('');
 
+}
 
+console.log(onlyVowels(stringWithVowels1));
+console.log(onlyVowels(stringWithVowels2));
 
 
 // --------------------5) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
-// Copy and paste or rewrite the function. Use the test variables provided below.
+// // Copy and paste or rewrite the function. Use the test variables provided below.
 
 var notAString1 = true
 var notAString2 = 42
 
+var vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+function removeVowels(str){
+if(!(typeof str === "string")){
+return "Not a string"
+}
+
+var withoutVowel = ""
+for(var i=0; i<str.length; i++){
+if(!vowels.includes(str[i])){
+withoutVowel = withoutVowel + str[i]
+}
+}
+return withoutVowel
+}
+
```

Comment:
This solution works great, but there are indentation issues. When everything is on the left margin, it's difficult to see the line of logic + where your function blocks begin and end.